### PR TITLE
test(crux-ui): image config, projects, teams, storage tests

### PR DESCRIPTION
**Pull request by ngyradi**
Added tests for: Dashboard, Image config, Projects, Storage, Teams.

### CODE REVIEW

File: web/crux-ui/e2e/utils/websocket-match.ts
Comment (robot9706) [inline web/crux-ui/e2e/utils/websocket-match.ts:9]: Only the `internalFromPort` is required?

File: web/crux-ui/e2e/utils/websocket-match.ts
Comment (robot9706) [inline web/crux-ui/e2e/utils/websocket-match.ts:65]: This should check all environment variables, not just the first.

File: web/crux-ui/e2e/utils/websocket-match.ts
Comment (robot9706) [inline web/crux-ui/e2e/utils/websocket-match.ts:90]: Same as environment.

File: web/crux-ui/e2e/utils/websocket-match.ts
Comment (robot9706) [inline web/crux-ui/e2e/utils/websocket-match.ts:115]: Same as environment.

File: web/crux-ui/e2e/with-login/dashboard.spec.ts
Comment (robot9706) [inline web/crux-ui/e2e/with-login/dashboard.spec.ts:11]: I think this should also look for the label. If there are multiple toggles on the profile page this won't work anymore.

File: web/crux-ui/e2e/with-login/version.spec.ts
Comment (robot9706) [inline web/crux-ui/e2e/with-login/version.spec.ts:36]: Comments?

File: web/crux-ui/e2e/with-login/storage.spec.ts
Comment (robot9706) [inline web/crux-ui/e2e/with-login/storage.spec.ts:58]: There's no need for `removeUrlDomainSuffix` to be async.

File: web/crux-ui/e2e/with-login/project.spec.ts
Comment (robot9706) [inline web/crux-ui/e2e/with-login/project.spec.ts:72]: Remove comment please.

File: web/crux-ui/e2e/with-login/nodes-deploy.spec.ts
Comment (robot9706) [inline web/crux-ui/e2e/with-login/nodes-deploy.spec.ts:15]: In this case there's no need for `IMAGE`, it can be replaced with `NGINX_TEST_IMAGE_WITH_TAG`.

**Review by robot9706 — APPROVED** [review]
Looks cool ✨, wait for another review and e2e.

**Comment by AmorfEvo** [comment]
LGTM (tests run with 4 workers)

**Review by robot9706 — APPROVED** [review]

**Review by AmorfEvo — APPROVED** [review]

**Pull request merged**
